### DISCUSSION TITLE
Multiple open Libraries

### DISCUSSION THREAD

**Discussion by leonuetzel** [flandreas/antares#465] [Q&A] [answered]
Ive seen that you closed the Issue with V1.0.
So i tried to create a Project that uses more than one Library, but i didnt manage to do so.
Can you explain please what i need to do?

**Comment by flandreas** [comment, marked as answer]
Open the project, right-click on the desktop node in the explorer tree, and execute „Add Library..“ in the popup-menu. Then choose the library to be added in the library dialog.

**Reply by leonuetzel** [reply]
Ah yeah, got it :)
Thank you
